doc/source/io.rst

```diff
@@ -662,8 +658,9 @@ If ``skip_blank_lines=False``, then ``read_csv`` will not ignore blank lines:
 
    .. ipython:: python
 
-      data = '# empty\n# second empty line\n# third empty' \
-                'line\nX,Y,Z\n1,2,3\nA,B,C\n1,2.,4.\n5.,NaN,10.0'
+      data = """# empty\n# second empty line\n# third empty \
+                line\nX,Y,Z\n1,2,3\nA,B,C\n1,2.,4.\n5.,NaN,10.0"""
```

Comment:
those lines are not equivalent

Reply:
I am unable to resolve an error here. On my local build the the second line is not appended to the variable data. I am quoting the code here. Need you help. 
`
In [68]: data = '# empty\n# second empty line\n# third empty' \

In [69]: 'line\nX,Y,Z\n1,2,3\nA,B,C\n1,2.,4.\n5.,NaN,10.0'
Out[69]: 'line\nX,Y,Z\n1,2,3\nA,B,C\n1,2.,4.\n5.,NaN,10.0'

In [70]: print(data)
\# empty
\# second empty line
\# third empty

In [71]: pd.read_csv(StringIO(data), comment='#', skiprows=4, header=1)
---------------------------------------------------------------------------
EmptyDataError                            Traceback (most recent call last)
<ipython-input-71-2ad3ec79b56f> in <module>
----> 1 pd.read_csv(StringIO(data), comment='#', skiprows=4, header=1)
`

Reply:
Note that if I copy paste these lines to a ipython terminal. It works well. I tried combination of spaces etc.

Reply:
@datapythonista - Can you please confirm the above? Otherwise everything is ready for a commit and push.

Reply:
Sorry, I missed that.

I didn't quite understand what's the problem, but checking it again, I think much better ways to write that are:
```python
data = '\n'.join(['# empty',
                  '# second empty line',
                  '# third empty',
                  'line',
                  'X,Y,Z',
                  '1,2,3',
                  'A,B,C',
                  '1,2.,4.',
                  '5.,NaN,10.0'])
```
or
```python
data = '''# empty
# second empty line
# third empty
line
X,Y,Z
1,2,3
A,B,C
1,2.,4.
5.,NaN,10.0'''
```

which makes possible to read the content